RISC-V_cva6/uart/uart.c

```diff
@@ -389,7 +392,9 @@ UART_set_rx_handler
 {
     //ASSERT(handler != INVALID_IRQ_HANDLER );
```

Comment:
should use configASSERT for these asserts.  Then the code goes away 'automatically' and we do not need to comment things out.